apply.js

```diff
@@ -1,26 +1,63 @@
-const jsPatches = ["displayFullName"]
-const cssPatches = ["hideWCAG", "alignDetailedGradesButton", "hideTutorsFromBoard"]
+const jsPatches = ["displayFullName"];
+const cssPatches = [
+  "hideWCAG",
+  "alignDetailedGradesButton",
+  "hideTutorsFromBoard",
+];
+
+const defaultOptions = {
+  ...cssPatches.reduce(
+    (acc, patch) => ({ ...acc, [patch]: { enable: true, type: "css" } }),
+    {}
+  ),
+  ...jsPatches.reduce(
+    (acc, patch) => ({ ...acc, [patch]: { enable: true, type: "js" } }),
+    {}
+  ),
+};
+
+chrome.storage.sync.set({ options: defaultOptions });
+
+let config = defaultOptions;
+
+chrome.storage.onChanged.addListener(function (changes, namespace) {
+  if (namespace !== "sync") return;
+  config = changes.options.newValue;
+  chrome.permissions.getAll((permissions) => {
+    const hostPatterns = permissions.origins || [];
+
+    hostPatterns.forEach((pattern) => {
+      chrome.tabs.query({ url: pattern }, (tabs) => {
+        tabs.forEach((tab) => {
+          chrome.tabs.reload(tab.id);
+        });
+      });
+    });
+  });
+});
 
 chrome.tabs.onUpdated.addListener((tabId, changeInfo, tab) => {
```

Comment:
There is one key thing missing, all files placed in the folders should be counted as one "module". We can handle that either here or by assigning multiple values to one checkbox while saving configuration.

I didn't mention stuff like this cuz I had no time for creating contribution guildlines, this project was created yesterday 😅, sorry but not including this would result in breaking everything that is already in-development features (3 branches in total)

Reply:
I don't understand what you mean by "all files placed in the folders should be counted as one "module"", so the patches folder is one module?